src/group/managed_group/mod.rs

```diff
@@ -956,24 +902,20 @@ impl ManagedGroup {
     /// Converts MlsPlaintext to MLSMessage. Depending on whether handshake
     /// message should be encrypted, MlsPlaintext messages are encrypted to
     /// MlsCiphertext first.
-    fn plaintext_to_mls_messages(
+    fn plaintext_to_mls_message(
         &mut self,
-        mut plaintext_messages: Vec<MlsPlaintext>,
-    ) -> Result<Vec<MlsMessage>, ManagedGroupError> {
-        let mut out = Vec::with_capacity(plaintext_messages.len());
-        for plaintext in plaintext_messages.drain(..) {
-            let msg = match self.configuration().handshake_message_format {
-                HandshakeMessageFormat::Plaintext => MlsMessage::Plaintext(plaintext),
-                HandshakeMessageFormat::Ciphertext => {
-                    let ciphertext = self
-                        .group
-                        .encrypt(plaintext, self.configuration().padding_size())?;
-                    MlsMessage::Ciphertext(ciphertext)
-                }
-            };
-            out.push(msg);
-        }
-        Ok(out)
+        plaintext: MlsPlaintext,
+    ) -> Result<MlsMessage, ManagedGroupError> {
+        let msg = match self.configuration().handshake_message_format {
+            HandshakeMessageFormat::Plaintext => MlsMessage::Plaintext(plaintext),
+            HandshakeMessageFormat::Ciphertext => {
```

Comment:
Codecov suggests that we never use this with `Ciphertext`. Can you check if this is really the case?

Reply:
Do you want to add a test (or parameterise some tests to use ciphertexts)? If not, please file a follow up.

Reply:
I've changed one of the managed group tests to use ciphertexts. That should increase the coverage.